doc/nrf/glossary.rst

```diff
@@ -459,8 +459,8 @@ Glossary
       See :ref:`zephyr:west`.
 
    West manifest file
-      The file containing the metadata of the manifest repository.
-      In the |NCS| and Zephyr, it is :file:`west.yml`.
+      The main file describing the contents of a west workspace, which is located in the manifest repository.
```

Comment:
Rationale: there are also other distinct metadata files, such as the YAML files that configure west extension commands.